[2025-02-09T15:40:32.283+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-09T15:40:32.314+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-09T15:40:27.880565+00:00 [queued]>
[2025-02-09T15:40:32.323+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-09T15:40:27.880565+00:00 [queued]>
[2025-02-09T15:40:32.325+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-09T15:40:32.533+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_directory> on 2025-02-09 15:40:27.880565+00:00
[2025-02-09T15:40:32.543+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=212) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-09T15:40:32.545+0000] {standard_task_runner.py:72} INFO - Started process 214 to run task
[2025-02-09T15:40:32.545+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'directory_and_file_verification_dag', 'check_directory', 'manual__2025-02-09T15:40:27.880565+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/my_directory_and_file_checker.py', '--cfg-path', '/tmp/tmp62v1i1yy']
[2025-02-09T15:40:32.546+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask check_directory
[2025-02-09T15:40:32.608+0000] {task_command.py:467} INFO - Running <TaskInstance: directory_and_file_verification_dag.check_directory manual__2025-02-09T15:40:27.880565+00:00 [running]> on host d4cc7d48468b
[2025-02-09T15:40:32.722+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='directory_and_file_verification_dag' AIRFLOW_CTX_TASK_ID='check_directory' AIRFLOW_CTX_EXECUTION_DATE='2025-02-09T15:40:27.880565+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-09T15:40:27.880565+00:00'
[2025-02-09T15:40:32.724+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-09T15:40:32.725+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-09T15:40:32.726+0000] {logging_mixin.py:190} INFO - Current task name:check_directory state:running start_date:2025-02-09 15:40:32.314741+00:00
[2025-02-09T15:40:32.726+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag and current dag run status:running
[2025-02-09T15:40:32.727+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-09T15:40:32.728+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-02-09T15:40:32.728+0000] {python.py:240} INFO - Done. Returned value was: check_file_in_directory
[2025-02-09T15:40:32.729+0000] {branch.py:38} INFO - Branch into check_file_in_directory
[2025-02-09T15:40:32.730+0000] {skipmixin.py:230} INFO - Following branch ('check_file_in_directory',)
[2025-02-09T15:40:32.748+0000] {skipmixin.py:278} INFO - Skipping tasks []
[2025-02-09T15:40:32.787+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-09T15:40:32.788+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=directory_and_file_verification_dag, task_id=check_directory, run_id=manual__2025-02-09T15:40:27.880565+00:00, execution_date=20250209T154027, start_date=20250209T154032, end_date=20250209T154032
[2025-02-09T15:40:32.803+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-09T15:40:32.804+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-09T15:40:32.804+0000] {logging_mixin.py:190} INFO - Dag name:directory_and_file_verification_dag queued_at:2025-02-09 15:40:27.910446+00:00
[2025-02-09T15:40:32.805+0000] {logging_mixin.py:190} INFO - Task hostname:d4cc7d48468b operator:BranchPythonOperator
[2025-02-09T15:40:32.841+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-09T15:40:32.881+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-09T15:40:32.884+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
